Tool/Sources/Preferences/Keys.swift

```diff
@@ -203,6 +203,10 @@ public extension UserDefaultPreferenceKeys {
     var codeiumEnterpriseMode: PreferenceKey<Bool> {
         .init(defaultValue: false, key: "CodeiumEnterpriseMode")
     }
+    
+    var codeiumChatEnabled: PreferenceKey<Bool> {
+        .init(defaultValue: true, key: "CodeiumChatEnabled")
```

Comment:
I don't think we need to allow users to disable chat. We can expose a knob to disable code lenses, but I think we can enable chat for everyone.